src/axolotl/core/trainer_builder.py

```diff
@@ -712,6 +712,22 @@ def build(self, total_num_steps):
             training_arguments_kwargs
         )
         training_arguments_kwargs["model_type"] = self.cfg.model_config_type
+
+        if self.cfg.noisy_embedding_alpha is not None:
+            # Deprecated, use neftune_noise_alpha
+            LOG.warning("noisy_embedding_alpha is deprecated, use neftune_noise_alpha")
+            training_arguments_kwargs[
+                "neftune_noise_alpha"
+            ] = self.cfg.noisy_embedding_alpha
+
+        if (
+            self.cfg.neftune_noise_alpha is not None
+            and self.cfg.neftune_noise_alpha > 0
```

Comment:
This validation > 0 should also be moved to `validate_config`

Reply:
Done.